src/main/kotlin/racingcar/domain/CarNames.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar.domain
+
+class CarNames(private val text: String) {
```

Comment:
text 변수를 프로퍼티로 선언하지 않아도 될것 같습니다 😄 
```suggestion
class CarNames(text: String) {
```

Reply:
이부분은 어떤식으로 개선해보면 좋을까요~?
프로퍼티로 선언하지 않는 방식이 어떤 방식인지 잘 모르겠습니다!
```kotlin
// AS-IS
class CarNames(text: String) {

// TO-BE
class CarNames(val text: String) {
```